### PR TITLE
Setup first mobile app 2: Adding about/contact me

### PR DESCRIPTION
**Pull request by mohisa302**
### Setup first mobile app
in this block:

- Added about me content
- added skills
- added contact form
- checked linters error

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -110,3 +112,487 @@ ul {
...
+  border: none;
+}
+
+.About-myself {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  background: #1c1a19;
+}
+
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [ ] Great job with your design :clap:. Kindly add the `background-image` in the about section as seen in your chosen Figma template 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 2]
```diff
@@ -110,3 +112,487 @@ ul {
...
+  align-self: center;
+  width: 80%;
+}
+
+.lang {
+  align-self: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [ ] Kindly align the languages to the center and space them as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: index.html [hunk 3]
```diff
@@ -46,8 +47,175 @@ <h2 class="job-title">I'm a Software Developer</h2>
...
+            <h1 class="about-text">About me</h1>
+            <p class="regular-text2">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+                 don’t hestiate to contact me.</p>
+             <button class="resume" type="button">Get My Resume</button>
+             <div class="line2"></div>
+             <div class="about-bar">
+                <nav class="bar">
+                    <div class="shape1"></div>
+                    <div class="titr">Language</div>
+                </nav>
```
Comment (Ugboaja-Uchechi) [inline index.html]: - [ ] Great job using a `ul` and `li` tag to group the list of languages 👍. According to the project requirements pease use `ul` and `li` for categories in "About myself" section. `Languages`, `Frameworks`, and `Skills` 🙏

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @mohisa302, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️

- [x] Great job using `README provided by Microverse kindly make it talk about your current project not the previous one :pray:

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (mohisa302) [reply style.css]: @Ugboaja-Uchechi Hello, thanks, for your review, can I use one image for my background?

File: index.html (hunk 3 above)
Comment (mohisa302) [reply index.html]: @Ugboaja-Uchechi sorry I didn't understand you. should I put Language in UI tag and ruby and HTML and ... in li tag and to this for skills and the others, or i should put the languages and skills and framework in li tags?

Comment (Ugboaja-Uchechi) [reply index.html]: Hi @mohisa302, Your code should be something like a nested list. Example:

```
<ul>
  <li>
    Languages
    <ul>
      <li>
        Ruby
      </li>
    </ul>
  </li>
</ul>
```

I hope it's clearer for you.

Comment (mohisa302) [reply index.html]: thanks so much

Comment (mohisa302) [reply index.html]: done

File: style.css (hunk 2 above)
Comment (mohisa302) [reply style.css]: done

File: style.css (hunk 1 above)
Comment (mohisa302) [reply style.css]: done

**Review by emmiiorji — APPROVED** [review]
Hi @mohisa302, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Adhered to GitHub flow
- Requested changes have been implemented

## Optional suggestions

- I suggest that you create a folder to contain the images in your website in order to reducer clutter of the home directory.
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**